horace_core/sqw/PixelData/@PixelDataBase/PixelDataBase.m

```diff
@@ -674,13 +678,24 @@
             %
             ids = obj.unique_run_id_;
         end
+        %
+        function do = get.keep_precision(obj)
+            do = obj.keep_precision_;
+        end
+        function obj = set.keep_precision(obj,val)
+            obj.keep_precision_ = logical(val);
+        end
+
     end
     %----------------------------------------------------------------------
     methods
         % return set of pixels, defined by its indexes
         pix_out = get_pixels(obj, abs_pix_indices,varargin);
-        %
+        %==================================================================
+        % Methods historically on pixels too, but useful mainly on sqw
+        % object
```

Comment:
I don't know what this means. What do you mean by historically? If it is to alert the reader to something that was done before but is no longer done now, it would be useful to be explicit. If something else, needs clarifying.

Reply:
It is still can be done, but I am not sure if it is any point in doing this. The methods exist on sqw and used and tested there, and here are left because they were there before. Do not know how to express it better.